app/src/main/java/com/orange/ouds/app/ui/components/link/LinkDemoScreen.kt

```diff
@@ -0,0 +1,167 @@
+/*
+ * Software Name: OUDS Android
+ * SPDX-FileCopyrightText: Copyright (c) Orange SA
+ * SPDX-License-Identifier: MIT
+ *
+ * This software is distributed under the MIT license,
+ * the text of which is available at https://opensource.org/license/MIT/
+ * or see the "LICENSE" file for more details.
+ *
+ * Software description: Android library of reusable graphical components
+ */
+
+package com.orange.ouds.app.ui.components.link
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.BoxScope
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.rememberBottomSheetScaffoldState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import com.orange.ouds.app.R
+import com.orange.ouds.app.ui.components.Component
+import com.orange.ouds.app.ui.utilities.composable.CustomizationBottomSheetScaffold
+import com.orange.ouds.app.ui.utilities.composable.CustomizationChoiceChipsColumn
+import com.orange.ouds.app.ui.utilities.composable.CustomizationSwitchListItem
+import com.orange.ouds.app.ui.utilities.composable.DemoScreen
+import com.orange.ouds.app.ui.utilities.composable.DetailScreenDescription
+import com.orange.ouds.core.component.coloredbox.OudsColoredBox
+import com.orange.ouds.core.component.link.OudsLink
+import com.orange.ouds.core.theme.OudsTheme
+import com.orange.ouds.core.theme.OudsThemeTweak
+import com.orange.ouds.core.utilities.OudsPreview
+import com.orange.ouds.foundation.utilities.UiModePreviews
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun LinkDemoScreen() = DemoScreen(rememberLinkDemoState()) {
+    CustomizationBottomSheetScaffold(
+        bottomSheetScaffoldState = rememberBottomSheetScaffoldState(),
+        bottomSheetContent = {
+            CustomizationSwitchListItem(
+                label = stringResource(R.string.app_common_enabled_label),
+                checked = enabled,
+                onCheckedChange = { enabled = it },
+            )
+            CustomizationSwitchListItem(
+                label = stringResource(R.string.app_components_common_onColoredBackground_label),
+                checked = onColoredBox,
+                onCheckedChange = { onColoredBox = it },
+            )
+            val sizes = remember { listOf(OudsLink.Size.Small, OudsLink.Size.Medium) }
```

Comment:
We could use `OudsLink.Size.entries` instead of a list. In that case, `remember` is not even mandatory.